### DISCUSSION TITLE
Feature: Click to Source

### DISCUSSION THREAD

**Discussion by arielvieira** [Code-Forge-Net/Remix-Dev-Tools#57] [Ideas]
Would be really cool if we could have the ability to open the source code of a element in the browser like Qwik and Svelte does.

Example in Qwik: https://youtu.be/e36y_HVRx5k
Example in Svelte: https://youtu.be/OuDSEvplXqY

**Comment by AlemTuzlak** [comment]
Already working on this, hopefully will be a part of next release

**Reply by AlemTuzlak** [reply]
this is included in the latest release @arielvieira you can CTRL + right-click to go to source in VS code

**Comment by AlemTuzlak** [comment]
Implemented!
